### PR TITLE
style fix for languages without white spaces

### PR DESCRIPTION
**Pull request by gera2ld**

### CODE REVIEW

File: src/options/views/tab-settings/index.vue [hunk 1]
```diff
@@ -38,22 +38,25 @@
             <option value="page">page</option>
             <option value="auto">auto</option>
           </select>
+          <a class="ml-1" href="https://violentmonkey.github.io/2018/11/23/inject-into-context/" target="_blank" rel="noopener noreferrer" v-text="i18n('learnInjectionMode')"></a>
         </label>
-        <a class="ml-1" href="https://violentmonkey.github.io/2018/11/23/inject-into-context/" target="_blank" rel="noopener noreferrer" v-text="i18n('learnInjectionMode')"></a>
       </div>
       <div class="mb-1 multi-opt-row">
```
Comment (tophf) [inline src/options/views/tab-settings/index.vue]: `.multi-opt-row` can be removed

**Comment by gera2ld** [comment]
`whitespace: 'condense'` does not work for me either.
image

**Comment by tophf** [comment]
Certainly works for me, here's my node_modules/@gera2ld/plaid-vue/webpack/vue.js:

```js
const VueLoaderPlugin = require('vue-loader/lib/plugin');

module.exports = (config, options) => {
  const {
    vueOptions,
  } = options;
  config.resolve.extensions = [
    ...config.resolve.extensions || [],
    '.vue',
  ];
  config.module.rules = [
    {
      test: /\.vue$/,
      loader: 'vue-loader',
      options: {
        compilerOptions: {
          whitespace: 'condense',
        },
      },
    },
    ...config.module.rules || [],
  ];
  config.plugins.push(new VueLoaderPlugin());
};
```

**Comment by gera2ld** [comment]
It's a bug in `plaid` and fixed in 1.5.2, `vueOptions` should not be `undefined`.

**Pull request merged**